gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/config/dsl/external/wuml/WUMLBaseListenerImpl.java

```diff
@@ -643,6 +643,30 @@ public void exitLogMediatorCall(WUMLParser.LogMediatorCallContext ctx) {
     }
 
     /**
+     * Handle custom mediator statements
+     */
+    @Override public void exitCustomMediatorCall(WUMLParser.CustomMediatorCallContext ctx) {
+        /* Proceed only of the configuration is valid */
+        if (ctx.getChildCount() == 7) {
```

Comment:
I couldn't figure out why it is 7 (not something else like 5). No problem if we have removed it in a later commit.

Thanks